datadotworld/datadotworld.py

```diff
@@ -182,9 +182,17 @@ def load_dataset(self, dataset_key, force_update=False, auto_update=False):
                 if (last_modified > datetime.utcfromtimestamp(
                         path.getmtime(str(descriptor_file)))):
                     if auto_update:
-                        backup_dir = path.join(self._config.cache_dir, owner_id,dataset_id, 'backup')
-                        move_cache_dir_to_backup_dir(backup_dir, cache_dir)
-                        descriptor_file = self.api_client.download_datapackage(dataset_key, cache_dir)
+                        try:
+                            backup_dir = path.join(self._config.cache_dir,
+                                                    owner_id,dataset_id,
+                                                    'backup')
+                            move_cache_dir_to_backup_dir(backup_dir,
+                                                         cache_dir)
+                            descriptor_file = self.api_client.download_datapackage(dataset_key, cache_dir)
+                        except RestApiError as e:
+                            shutil.move(backup_dir, cache_dir)
```

Comment:
Why would you treat this exception differently from https://github.com/datadotworld/data.world-py/pull/80/files#diff-31fc69a07d5fac6aa1efbc356d841e14R171?

Reply:
Right, I skipped the if-check in there.